src/test/resources/features/FTUX/Favorites.feature

```diff
@@ -132,6 +132,22 @@ Feature: Favorite activities
     And The activity is not displayed in the search 'activityFavs_US06_02_1'
     And The activity is not displayed in the search 'activityFavs_US06_02_0'
 
+
+  Scenario: SpaceFav_US01.11: Bookmark a space as a favorite (space page)
+    Given I am authenticated as admin
+    And I create the first random user if not existing
+    And I create a random space
+    And I connect with the first created user
+    And I go to the random space
+    And I 'Join'
+    When I go to spaces page
+    And I search for the created space
```

Comment:
Try to reuse existing scenario definition :
https://github.com/Meeds-io/meeds-qa-ui/blob/c01e8c4626e2a571abc351cdedaf9d51f9000622/src/test/java/io/meeds/qa/ui/steps/definition/ManageSpaceStepDefinitions.java#L344-L347